docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+
+services:
+  script:
```

Comment:
add "image" parameter so when the docker builds the image from the dockerfile it will have an informative name